app/views/votes/_toggle_like.html.erb

```diff
@@ -3,8 +3,15 @@
   liked = user_signed_in? && current_user.voted_for?(votable)
 %>
 
-<%= form_tag vote_path, method: (liked ? :delete : :post) do %>
+<%= form_tag vote_path, method: (liked ? :delete : :post), class: "like-form" do %>
```

Comment:
I agree with this, I think we'll have better visibility on this when we make changes based on react